### PR TITLE
Limit query length to be 10 words at most

### PR DESCRIPTION
**Pull request by chaconnewu**
issue link:
https://github.com/commonsearch/cosr-front/issues/17

Screenshot:
10 words

### CODE REVIEW

**Comment by sylvinus** [comment]
Wow that's great, thanks @chaconnewu!

My only feedback is that it would be nice to have this limit configurable, by adding an entry in `config.go`. Seems doable?

**Comment by chaconnewu** [comment]
@sylvinus great suggestion! Just pushed a commit for it.

File: server/config.go [hunk 1]
```diff
@@ -42,6 +42,9 @@ type ConfigSpec struct {
 
 	// ResultPageSize controls the number of results on each page.
 	ResultPageSize int `default:"25"`
+
+	// Maximum allowed number of words for a query
+	MaxQueryLengh int `default:"10"`
```
Comment (sylvinus) [inline server/config.go]: `MaxQueryLength` ? Or maybe `MaxQueryTerms` would be better?

**Comment by chaconnewu** [comment]
Just renamed the variable and fixed the comment lint error

**Comment by sylvinus** [comment]
Awesome! Congrats, probably the cleanest patch we've had so far ;)

**Comment by chaconnewu** [comment]
Thanks @sylvinus ! It is a lot of pleasure to work on this project  😄

**Pull request merged**